solutions/swb-ui/infrastructure/src/hosting-infra/ecs-cluster.ts

```diff
@@ -11,6 +11,7 @@ import {
   ApplicationListener,
   ApplicationTargetGroup,
   ListenerCondition,
+  Protocol,
```

Comment:
may be apply cfn-nag and address the warnings and errors ?

Reply:
No new CFN nag errors introduced in these changes. Existing ones are going to be addressed in the larger epic in our backlog.